index.html

```diff
@@ -1348,6 +1348,19 @@ <h2>
 MAY be encoded in JSON Web Key (JWK) format using the <code>publicKeyJwk</code>
 property. Both the raw 32-byte base58-btc format and JWK format MUST be
 supported by DID Document Processors. The use of other key formats for RSA MUST
+cause the DID Document Processor to raise an error.
+            </td>
+          </tr>
+          <tr>
+            <td>
+ed25519
+            </td>
+            <td>
+ed25519 key values MAY be expressed as the raw 32-byte value encoded in
+base58-btc format using the <code>publicKeyBase58</code> property or
+MAY be encoded in JSON Web Key (JWK) format using the <code>publicKeyJwk</code>
+property. Both the raw 32-byte base58-btc format and JWK format MUST be
```

Comment:
same comment here. s/base58-btc/base58/

Reply:
There are two base58 encodings... Bitcoin's and Flickr's... We need to be specific about which one we're using (most use Bitcoin's base58 encoding scheme).

Reply:
Tried to clarify in Fixed in 41b0b4781bab8fdfae2cc42990bacc670342e04f.